src/Compilers/CSharp/Test/Semantic/Semantics/CollectionLiteralTests.cs

```diff
@@ -1173,7 +1173,6 @@ static void Main()
                     }
                 }
                 """;
-            // PROTOTYPE: Should compile and run successfully: expectedOutput: "[1, 2, 3], "
```

Comment:
>PROTOTYPE [](http://example.com/codeflow?start=15&length=9)

Natural type for collection literals is not supported, so `[x]` and `[y]` are target-typed only.